src/styles/styles.css

```diff
@@ -540,6 +548,12 @@ a.button-link.next {
 
 a.button-link.descriptive {
   flex-direction: column;
+  color: #1A182A;
```

Comment:
Maybe it's just me, but these buttons being the only ones that go grey->grey seems a little odd to me. What do you think about them being the same as button-link.next?